src/components/admin/Tables/ActionTable.js

```diff
@@ -12,30 +12,44 @@ import TableFooter from '@material-ui/core/TableFooter';
 import TablePagination from '@material-ui/core/TablePagination';
 import IconButton from '@material-ui/core/IconButton';
 
+import {
+  AppBar,
+  Tabs,
+  Tab,
+} from '@material-ui/core';
+
 import styles from '../admin.module.css';
 import {TablePaginationActions,
   computeEmptyRows,
   computeRows} from '../TablePaginationActions';
 
 import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
 import {Typography} from '@material-ui/core';
-
+import type {ActionType} from '../FlowTypes.js';
+import {fieldQuery, moveSolvedAction} from '../../database/Queries.js';
 import ActionInfo from '../Dialogs/ActionInfo';
 
-const rows: Array<any> = Array.from([
-  '[NOOGLER CHECK 1]: A partner of a noogler requires access',
-  '[DATABASE]: A new member could not be added to the database.',
-  '[NOOGLER CHECK 2]: A partner of a noogler requires access',
-], (message: string, id: number): {id: number, message: string, date: any} => ({
-  id,
-  message,
-  date: new Date(),
-}));
-
 export default function ActionTable(): React.Node {
   const [page, setPage] = React.useState(0);
   const [rowsPerPage, setRowsPerPage] = React.useState(5);
   const [selectedRow, setSelectedRow] = React.useState(-1);
+  const [rows, setRows] = React.useState([]);
+  const [tab, setTab] = React.useState(0);
+
+  React.useEffect(() => {
+    (async () => {
+      let newRows = [];
+      if (tab === 0) {
+        const start = page * rowsPerPage + 1;
+        newRows = await fieldQuery('actions', 'count', start, rowsPerPage);
+      } else {
+        const start = page * rowsPerPage + 1;
```

Comment:
nit: `const start` is the same as in the "tab === 0" case. Would it make sense to pull it to the enclosing scope?